core/src/types.ts

```diff
@@ -266,3 +266,53 @@ export declare interface DebtAuctionEnriched extends DebtAuctionActive {
 }
 
 export declare interface DebtAuctionTransaction extends DebtAuctionEnriched, CompensationAuctionTransactionFees {}
+
+type CollateralType = CollateralConfig['title'];
+
+export declare interface LiquidationLimits {
+    maximumProtocolDebtDai: BigNumber;
+    maximumCollateralDebtDai: BigNumber;
+    currentProtocolDebtDai: BigNumber;
+    currentCollateralDebtDai: BigNumber;
+}
+
+export declare interface VaultCollateralParameters extends LiquidationLimits {
+    stabilityFeeRate: BigNumber;
+    maxDebtPerCollateralUnit: BigNumber;
+}
+
+export declare interface VaultBase {
+    id: number;
+    address: string;
+    collateralType: CollateralType;
```

Comment:
I would also add `lastSyncedAt: Date` and `network: string` keys somewhere in those vault types